### PR TITLE
Wrapped errors with xerrors

### PR DESCRIPTION
**Pull request by jBugman**
For future compatibility ([reference](https://github.com/golang/go/wiki/ErrorValueFAQ))

### CODE REVIEW

File: messenger.go [hunk 1]
```diff
@@ -192,11 +194,11 @@ func (m *Messenger) ProfileByID(id int64, profileFields []string) (Profile, erro
 		qr := QueryResponse{}
 		err = json.Unmarshal(content, &qr)
 		if qr.Error != nil {
-			err = fmt.Errorf("facebook error: %s", qr.Error.Message)
+			return p, xerrors.Errorf("facebook error: %w", qr.Error)
```
Comment (paked) [inline messenger.go]: IMO you should not change the way the control flow works here: while it's clever, it is a bit harder to follow and doesn't really add anything over the way it was done before.

**Comment by paked** [comment]
Apart from my one comment this seems fine.

Thanks for the PR!

File: messenger.go (hunk 1 above)
Comment (jBugman) [reply messenger.go]: Changed it back

**Comment by paked** [comment]
Thanks for the PR!

**Pull request merged**